frontend/src/testbreed_form.tsx

```diff
@@ -182,7 +210,29 @@ export function InbreedingForm() {
               );
             })}
         </div>
+        <div className={style.lineBreak}></div>
         <form className={style.form}>
+          <MuiPickersUtilsProvider utils={DateFnsUtils}>
```

Comment:
Layout wise, it would be nice if the picker was less wide. On a full screen it's hard to even see the calender icon in the right end of the picker at a first glance.

Reply:
Yes agree.  Could we make this setting hidden and have a gear icon like "advance" settings or similar